### ISSUE TITLE
Template. aria-required is missing

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#67]
### Description of issue

When adding a new folder in the templates directory, the field is required but is missing aria-required="true"

![image](https://user-images.githubusercontent.com/5230720/136038960-0505eee8-93c7-4a4f-b205-9ed261196fd1.png)

### Potential fix

Adding `aria-required="true"` to the input field will fix this issue

### Resources
https://axe.deque.com/rule-help/en/aria-required-missing
